Reference/Templating/Mvc/views.md

```diff
@@ -10,82 +10,56 @@ _Working with MVC Views and Razor syntax in Umbraco_
 
 ## Properties available in Views
 
-All Umbraco views inherit from `Umbraco.Web.Mvc.UmbracoTemplatePage` which exposes many properties that are available in razor:
+All Umbraco views inherit from `Umbraco.Web.Mvc.UmbracoViewPage<ContentModels.NameOfYourDocType>` along with the using statement `@using ContentModels = Umbraco.Web.PublishedModels;`. This exposes many properties that are available in razor,. This means that properties on the Document Type can be accessed in a number of ways:
 
+* @Model (of type `Umbraco.Web.Mvc.RenderModel`) -> the model for the view which contains the standard list of IPublishedContent properties but also gives you access to the typed current page (of type whatever type you have added in the angled brackets).
```

Comment:
I think RenderModel has become ContentModel... but also I think in using UmbracoViewPage the Model is IPublishedContent?

Reply:
If you type the model in the partial with angle brackets like this

@inherits UmbracoViewPage<SiteSettings>

You get the model returned as it is in the brackets

Reply:
I've flipped these now and added a little more info - do we think it's resolved? It might be that there's still more to do.